lib/docker2aci.go

```diff
@@ -447,6 +442,20 @@ func generateManifest(layerData DockerImageData, dockerURL *ParsedDockerURL) (*s
 	return genManifest, nil
 }
 
+func getExecCommand(entrypoint []string, cmd []string) types.Exec {
+	var command []string
+	if entrypoint == nil && cmd == nil {
+		return nil
+	}
+	command = append(entrypoint, cmd...)
+	// non-absolute paths are not allowed, fallback to "/bin/sh -c command"
```

Comment:
We could warn with something like `Exec is not an absolute path, using "/bin/sh -c EXEC". The resulting ACI could be non-functional`.

This will be printed for each layer that we convert so it might be too verbose, especially if we're squashing. We could accumulate the warnings and print only one of each at the end. Something like:

```
Warnings found when converting the image. The resulting ACI could be non-fuctional:
        Exec is not an absolute path, using "/bin/sh -c EXEC"
        Docker User is empty, assuming root
```